corehq/apps/change_feed/data_sources.py

```diff
@@ -11,7 +12,7 @@
 FORM_SQL = 'form-sql'
 CASE_SQL = 'case-sql'
 SMS = 'sms'
-LEDGER = 'ledger'
+LEDGER_NEW = 'ledger-new'
```

Comment:
thoughts on making this `ledger-v2` or `ledger-sql`? just thinking about a few years from now when we could need `newer`...